src/main/webapp/feed.html

```diff
@@ -34,6 +34,16 @@ <h2> Play Even Harder </h2>
     <div id="content" class="container">
       <h1>Message Feed</h1>
       <hr />
+      <div id="feed-create-container">
```

Comment:
What is the use case of this?

Reply:
for the UI, I added a creation form for creating new post at the post feed page

Reply:
So, is it supposed to works like form from `User-Page.html`?